radix-engine-tests/tests/metering.rs

```diff
@@ -67,17 +67,17 @@ fn test_basic_transfer() {
     // Or you can run just this test with the below:
     // cargo test -p radix-engine-tests --test metering -- test_basic_transfer
     assert_eq!(
-        42500 /* CreateNode */
-        + 83000 /* DropLock */
+        40000 /* CreateNode */
+        + 75500 /* DropLock */
         + 40000 /* DropNode */
         + 10970 /* Invoke */
-        + 84000 /* LockSubstate */
-        + 176780 /* ReadSubstate */
+        + 76500 /* LockSubstate */
+        + 169570 /* ReadSubstate */
         + 62500 /* RunNative */
         + 7500 /* RunSystem */
-        + 100000 /* TxBaseCost */
-        + 2520 /* TxPayloadCost */
-        + 200000 /* TxSignatureVerification */
+        + 50000 /* TxBaseCost */
```

Comment:
All these tx costs have halved? Was there a bug before where we charged twice?

Reply:
Yup, this was a bug!